plugins/scanners/dos/src/main/kotlin/DosScanner.kt

```diff
@@ -187,18 +188,18 @@ class DosScanner internal constructor(
             return null
         }
 
-        val jobResponse = client.addScanJob(zipName, purls)
+        val jobResponse = client.addScanJob(zipName, dosPackages)
         val id = jobResponse?.scannerJobId
 
         if (id == null) {
-            issues += createAndLogIssue(name, "Failed to add scan job for '$zipName' and $purls.")
+            issues += createAndLogIssue(name, "Failed to add scan job for '$zipName' and $dosPackages.")
```

Comment:
The `PackageInfo.toString()` output may not be best suited for the issue message. 
Would it make sense to keep on logging purls only?